.changeset/modern-moons-pick.md

```diff
@@ -0,0 +1,7 @@
+---
+'fingerprint-pro-server-api-openapi': minor
+---
+
+**events**: Adds support for proxy detection confidence, both for residential and public web proxies.
+
+**webhook**: Adds support for proxy detection confidence, both for residential and public web proxies.
```

Comment:
Changes to Smart signal schemas affect multiple scopes (`events`, `events-search`, `webhook`). We don't have strong conventions here yet, but so far we have usually just left out the scope in the changeset file, like this.

```suggestion
---
'fingerprint-pro-server-api-openapi': minor
---

Add `confidence` property to the Proxy detection Smart Signal, which now supports both residential and public web proxies.
```

Reply:
@ilfa The CLI leads users to the wrong path here, since it only offers a choice of a single scope. Should we adjust the CLI script to also offer `smart-signals` as a scope, or `no scope`, or something similar?